scenarioo-server/build.gradle

```diff
@@ -24,6 +24,47 @@ apply plugin: 'eclipse-wtp'
 apply from: "${rootProject.projectDir}/build-java.gradle"
 apply plugin: 'org.springframework.boot'
 
+apply plugin: 'cz.habarta.typescript-generator'
+buildscript {
```

Comment:
Buildscript usually goes before applying any plugins as far as I know.

Reply:
Does not matter.